spec/services/reports/children_served_report_service_spec.rb

```diff
@@ -42,6 +42,7 @@
         create_list(:line_item, 5, :distribution, quantity: 200, item: disposable_item, itemizable: dist)
         create_list(:line_item, 5, :distribution, quantity: 300, item: non_disposable_item, itemizable: dist)
       end
+      # within_time total distributed i
```

Comment:
accidental commit?

Reply:
fixed in commit [94030f7](https://github.com/rubyforgood/human-essentials/pull/4665/commits/94030f75aefe8cfa6ecc4e6155da4c8e18bb630f)